src/components/Fields/TextArea.scss

```diff
@@ -107,9 +87,41 @@ $textFieldLabelColor: $charcoal !default;
         width: 100%;
     }
 
+    &-validation-message {
+        margin-top: 8px;
+        font-family: "DINOT", Arial, sans-serif;
+        font-size: 11px;
+        font-weight: normal;
+        color: $pink2;
+        letter-spacing: .92px;
```

Comment:
Does this letter spacing work? `.92px` seems like a suspicious value

Reply:
Yes, it works. This property was defined per design. There's a subtle difference.

**With .92px letter-spacing**
![image](https://user-images.githubusercontent.com/348776/68606097-713d5980-048c-11ea-9f80-35c236383a3d.png)

**Without letter-spacing defined**
![image](https://user-images.githubusercontent.com/348776/68606080-65519780-048c-11ea-8769-8b8dc5338e90.png)